examples/alloydb-vectorstore-example/alloydb_vectorstore_example.go

```diff
@@ -122,21 +121,7 @@ func main() {
 	}
 
 	// Create a new alloydb vectorstore .
-
-	vs, err := alloydb.NewVectorStore(ctx, pgEngine, e, table,
-		alloydb.WithMetadataColumns(
-			[]alloydbutil.Column{
-				alloydbutil.Column{
-					Name:     "area",
-					DataType: "int",
-					Nullable: false,
-				},
-				alloydbutil.Column{
-					Name:     "population",
-					DataType: "int",
-					Nullable: false,
-				},
-			}))
+	vs, err := alloydb.NewVectorStore(ctx, pgEngine, e, table)
```

Comment:
Sorry my comments have not been clear:
* init table - should take in metadata columns using []alloydbutil.Column
* vector store - should take in metadata columns as string names
We want to set up the vector store to use the metadata column names here

Reply:
done!